wandb/sdk/data_types/Html.py

```diff
@@ -0,0 +1,118 @@
+import os
+from typing import Sequence, Type, TYPE_CHECKING, Union
+
+import six
```

Comment:
I don't think we need this one anymore... since we don't support py2.7

Reply:
fixed in next commit